test/test_admin.py

```diff
@@ -0,0 +1,8 @@
+
+def test_admin_pagination(client):
+    res=client.post(
+      "/api/v1/admin/users/"
+    )
+    assert res.json().status_code == 202
+    assert res.json()
```

Comment:
Your test should check for an expected record you know is there.
You can create a user for instance, and then try fetching and asserting that the name of the first user item matches the one you created.